src/stylus/fablevision/trial-count.styl

```diff
@@ -0,0 +1,25 @@
+.geniblocks.trial-count
+  width: 158px
+  height: 107px
+  background-image: url('../resources/fablevision/huds/Trial_screen.png')
+  background-size: 100% 100%
+
+  .text-area
```

Comment:
I agree it's not always obvious. In this case I was imagining that there might be other HUD text areas for which similar rules might apply in other HUD elements. Perhaps `hud-text-area` would make more sense? I think we can leave it for now until we see whether others arise.